src/services/scheduler.ts

```diff
@@ -0,0 +1,48 @@
+import EventEmitter from 'events';
+import { run } from 'graphile-worker';
+
+import * as dbMan from './db-manager'
+import * as iasql from '../services/iasql'
+import * as logger from '../services/logger'
+
+const workerShutdownEmitter = new EventEmitter();
+
+// graphile-worker here functions as a library, not a child process.
+// It manages its own database schema
+// (graphile_worker) and migrations in each user db using our credentials
+export async function start(dbAlias: string, dbId: string, user: any) {
+  // Run a worker to execute jobs:
+  const runner = await run({
+    // TODO pass a connection pool here instead of telling it to create a new one
+    connectionString: dbMan.ourPgUrl(dbId),
```

Comment:
if it is being started with out connection string, how it will `apply` users changes?

Reply:
yeah, I'm removing that comment since the connection that starts the scheduler dies with the request so it doesn't make sense